### DISCUSSION TITLE
How to select an approperiate validation set (for train) and test set (for test)?

### DISCUSSION THREAD

**Discussion by Terrla** [deepmodeling/deepmd-kit#1290] [Q&A]
Dear DP developers,
       I am confused about the data selection between the validation set in the **training** and the input data in the **test** process. What is the difference between validation and test data selection?
       For example, I have DFT data of the same system at 700 K, 800 K, and 1000 K, and I have set the data of 700 K and 1000 K as input trained data (extract part of the data as a validation set). After freezing the model, can I set the data of 800 K as input for training? Or,  should I set the same data (700 K and 1000 K) as input of the test?  @njzjz 

Kind Regards,
Terrla

**Comment by njzjz** [comment]
If you are using the same or similar data for testing, you can see the training effectivity, and it's expected to see an accurate result. If you are using a different data, you can see the transferability and robustness of the model and it doesn't work in all time.

**Reply by Terrla** [reply]
Thanks for ur time.  I am wondering what fluctuation range of force fluctuation is considered accurate?

**Reply by njzjz** [reply]
There isn't an absolute criterion as it varies with different systems.

**Comment by LiangMD-BGI** [comment]
I have a quick question: Does using a validation set help converge the training (i.e., achieving smaller RMSE values for energy, forces)?

**Reply by njzjz** [reply]
The validation set has no effect on training.
